### PR TITLE
CART-39 added complaint popup

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/Complaint/config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+'use strict';
```
Comment (vetalcore) [inline dev/publicDev/Complaint/config.js:1]: wrap into IIFE block with passed params

File: dev/publicDev/Complaint/ctrl.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+"use strict";
+
+(() => {
```
Comment (vetalcore) [inline dev/publicDev/Complaint/ctrl.js]: arrow functions dont work in FF etc, replace with function() {} ...

File: dev/publicDev/Complaint/ctrl.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+"use strict";
```
Comment (vetalcore) [inline dev/publicDev/Complaint/ctrl.js]: follow the same convention for file names (complaintController or complaint-controller)...

File: dev/publicDev/Complaint/ctrl.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  complaintCtrl.$inject = ['complaintService', '$location'];
+
+  function complaintCtrl(complaintService, $location) {
+    var vm = this;
+
+    vm.save = function(form) {
+      if (form.$valid) {
+        complaintService.send(vm.data)
+          .then(function(res) {
+            console.log(res.data);
```
Comment (vetalcore) [inline dev/publicDev/Complaint/ctrl.js]: remove console statement

File: dev/publicDev/Complaint/service.js [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+  function complaintService($http) {
+    return {
+      send: send
+    }
+
+    function send(data) {
+      return $http.post('api/complaint', data);
+    }
+  }
+})();
```
Comment (vetalcore) [inline dev/publicDev/Complaint/service.js]: pass angular as param

File: dev/publicDev/Complaint/style.scss [hunk 6]
```diff
@@ -0,0 +1,130 @@
...
+      }
+
+      .title {
+        display: none;
+      }
+
+      .complaint-form {
+        padding-top: 20px;
+
+        .name,
```
Comment (vetalcore) [inline dev/publicDev/Complaint/style.scss]: group with some general class name

File: dev/publicDev/Complaint/style.scss [hunk 7]
```diff
@@ -0,0 +1,130 @@
+.complaint {
```
Comment (vetalcore) [inline dev/publicDev/Complaint/style.scss:1]: try to use bootstrap

**Review by vetalcore — CHANGES_REQUESTED** [review]
resolve review changes

File: dev/publicDev/Complaint/config.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/config.js:1]: fixed

File: dev/publicDev/Complaint/ctrl.js (hunk 3 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/ctrl.js]: fixed

File: dev/publicDev/Complaint/ctrl.js (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/ctrl.js]: fixed

File: dev/publicDev/Complaint/ctrl.js (hunk 4 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/ctrl.js]: fixed

File: dev/publicDev/Complaint/service.js (hunk 5 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/service.js]: fixed

File: dev/publicDev/Complaint/style.scss (hunk 6 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/style.scss]: fixed

File: dev/publicDev/Complaint/style.scss (hunk 7 above)
Comment (RomanFedytskyi) [reply dev/publicDev/Complaint/style.scss:1]: fixed

**Review by vetalcore — APPROVED** [review]

**Pull request merged**